packages/stateful/voting-module-adapter/adapters/CwdVotingCw721Staked/hooks/useProfileNewProposalCardAddresses.ts

```diff
@@ -0,0 +1,24 @@
+import { useTranslation } from 'react-i18next'
+
+import { ProfileNewProposalCardAddress } from '@dao-dao/types'
+
+import { useGovernanceTokenInfo } from './useGovernanceTokenInfo'
+
+export const useProfileNewProposalCardAddresses =
+  (): ProfileNewProposalCardAddress[] => {
+    const { t } = useTranslation()
+
+    const { stakingContractAddress, governanceTokenAddress } =
+      useGovernanceTokenInfo()
+
+    return [
+      {
+        label: t('info.govTokenAddress'),
```

Comment:
we may want to use verbiage other than 'governance token'. probably 'nft collection'? wdyt?

Reply:
@NoahSaso hmmm...i could go either way on this b/c using gov token as the label keeps it in line with cw20 and native daos, but i do like being more descriptive because for a while as a user, it took me a while to understand that the governance token was the cw20/native token/cw721